aspnetcore/security/enforcing-ssl.md

```diff
@@ -1,18 +1,46 @@
 ---
-title: 🔧 Enforcing SSL | Microsoft Docs
+title: Enforcing SSL in an ASP.NET Core app | Microsoft Docs
 author: rick-anderson
-description: 
-keywords: ASP.NET Core,
+description: Shows how to require SSL in a web app and how to set IIS Express to use SSL
+keywords: ASP.NET Core, SSL, HTTPS, RequireHttpsAttribute, IIS Express
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 03/19/2017
 ms.topic: article
 ms.assetid: 4694e563-e91a-4ecd-b7ed-00b3f1eee2b5
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: security/enforcing-ssl
 ---
-# 🔧 Enforcing SSL
+# Enforcing SSL in an ASP.NET Core app
 
-> [!NOTE]
-> We are currently working on this topic.We welcome your input to help shape the scope and approach. You can track the status and provide input on this [issue](https://github.com/aspnet/Docs/issues/135) at GitHub.If you would like to review early drafts and outlines of this topic, please leave a note with your contact information in the [issue](https://github.com/aspnet/Docs/issues/135).Learn more about how you can [contribute](https://github.com/aspnet/Docs/blob/master/CONTRIBUTING.md) on GitHub.
+This document shows how to:
+
+- Require all requests use SSL (that is are HTTPS requests).
+- Redirect all HTTP requests to HTTPS.
+- Set up IIS Express to use SSL/HTTPS.
+
+## Require SSL
+
+The [RequireHttpsAttribute](https://docs.microsoft.com/aspnet/core/api/microsoft.aspnetcore.mvc.requirehttpsattribute) is used to require request use HTTPS. You can decorate controllers or methods with this attribute or you can apply it globally as shown below:
+
+* Add the following code to `ConfigureServices` in `Startup`:
+
+[!code-csharp[Main](authentication/accconfirm/sample/WebApp1/Startup.cs?name=snippet2&highlight=4-)]
```

Comment:
Using snippet code from a different article's samples folder seems likely to cause maintenance issues when someone changes the sample and doesn't realize it's used elsewhere.  Consider adding comments in the sample project warning that the code is referenced elsewhere.  Consider using the Common folder which already has a project that might need minor updates for this article.